mesonbuild/backend/xcodebackend.py

```diff
@@ -147,6 +151,17 @@ def __init__(self, key: str, value: T.Union[PbxArray, PbxDict, str, int], commen
         else:
             self.comment = comment
 
+    def quote_value(self, value: str) -> str:
+        quoted = f'"{value}"'
+
+        if len(value) == 0:
```

Comment:
```suggestion
        if not value:
```

checking length of 0 is an antipattern in python, since it's considerably slower than calling the bool check.